pkgs/top-level/all-packages.nix

```diff
@@ -1579,6 +1579,10 @@ with pkgs;
 
   asynk = callPackage ../tools/networking/asynk { };
 
+  b2sum = callPackage ../tools/security/b2sum {
```

Comment:
We should probably move [tthsum](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/tthsum/default.nix) from ../tools/misc to ../tools/security then, for conformance.

Reply:
I judged by `tthsum` and `rhash` and decided the latter was placed more appropriately, so, yes, I think, we should.